src/file_pipe_log/pipe_builder.rs

```diff
@@ -429,10 +429,13 @@ impl<F: FileSystem> DualPipesBuilder<F> {
     fn initialize_files(&mut self) -> Result<()> {
         let now = Instant::now();
         let target_file_size = self.cfg.target_file_size.0 as usize;
-        let mut target = self
```

Comment:
Return early if `!prefill_for_recycle`.

Reply:
Not recommended. 
When `!prefill_for_recycle`, it should check whether there still exists remained `recycled_files` and clear them if exists. And by the way, the later processing when `!prefill_for_recycle` just do several checking jobs, not costing much on CPU or other resources.